Python/Product/PythonTools/PythonTools/Navigation/CodeWindowManager.cs

```diff
@@ -143,8 +145,8 @@ class CodeWindowManager : IVsCodeWindowManager, IVsCodeWindowEvents {
             var wpfTextView = VsEditorAdaptersFactoryService.GetWpfTextView(vsTextView);
             if (wpfTextView != null) {
                 var factory = ComponentModel.GetService<IEditorOperationsFactoryService>();
-                var editFilter = new EditFilter(wpfTextView, factory.GetEditorOperations(wpfTextView), _serviceProvider);
-                editFilter.AttachKeyboardFilter(vsTextView);
+                EditFilter.GetOrCreate(_serviceProvider, ComponentModel, vsTextView);
+                ReplEditFilter.GetOrCreate(_serviceProvider, ComponentModel, vsTextView);
```

Comment:
I'm pretty sure we still don't need this to be created here...  We already still attach this in ReplWindowCreationListener and the CodeWindowManager is never used for a REPL window